spec/system/donation_system_spec.rb

```diff
@@ -607,6 +607,56 @@
         total_quantity = find("#donation_quantity").text
         expect(total_quantity).to eq "0 (Total)"
       end
+
+      context "when an finalized audit has been performed on the donated items" do
```

Comment:
All 3 of these tests should be request tests, not system tests. They're not interactive, so there's no reason to run them in a browser.

Reply:
@dorner convert the system specs into request specs.